src/node_task_runner.cc

```diff
@@ -253,7 +256,7 @@ void RunTask(std::shared_ptr<InitializationResultImpl> result,
   auto package_json = FindPackageJson(cwd);
 
   if (!package_json.has_value()) {
-    fprintf(stderr, "Can't read package.json\n");
+    fprintf(stderr, "Can't find package.json for directory %s\n", cwd.string().c_str());
```

Comment:
There are other changes similar to this in the PR. Can you replace them as well?

```suggestion
    fprintf(stderr, "Can't find package.json for directory %s\n", cwd.c_str());
```